tests/integration_tests/integration_settings.py

```diff
@@ -39,6 +39,17 @@
 # to this format internally; in this case, to "None::ubuntu::focal::20.04".)
 OS_IMAGE = "focal"
 
+
+# Determines unique image type or flavor to exercise if the cloud supports
+# image-type lookup for daily_image and released_images.
+#
+# One of the following:
+# - generic
```

Comment:
Where do these names come from? I assume that this is a pycloudlib construct? Can we please include a pointer to wherever in pycloudlib this is documented to ease future maintenance of this?

Reply:
These come from the ImageType enum in pycloudlib. (`pycloudlib.cloud.ImageType`)